content/about/about_people.rst

```diff
@@ -337,3 +337,38 @@ runnerboi
 .. raw:: html
 
   <br/>
+
+`Yuning Gao`_
+----------------
+
+.. image:: /images/yuning.jpg
+    :width: 110px
+    :align: right
+    :alt: Yuning Gao
+
+.. class:: no-breaks
+
+  **Media & Events Coordinator/Writer**
+
+Yuning joined the OSL in April 2019 and is studying Psychology, Computer Science and Marketing.
+She first learned about the OSL during a Marketing related job searching.
+Yuning joined the lab to contribute to the open source community as well as to gain
+knowledge about social media management and event planning.
+In her free time, she enjoys Kendo, watch movies, and camping with hammocks.
```

Comment:
```suggestion
In her free time, she enjoys Kendo, watching movies, and camping with hammocks.
```
I forgot to suggest this in the last review. Do you think this should be in `V-ing` form?